src/lib/exeRunner.ts

```diff
@@ -51,19 +46,27 @@ export class ExeRunner {
             const stderr = new Array<string>();
 
             this.logger.info(`exe: ${this._exePath}, first arg of ${args.length}: ${args.length ? args[0]: '<none>'}`);
-            const pac = spawn(this._exePath, args, { cwd: this.workingDir });
+            const process = spawn(this._exePath, args, { cwd: this.workingDir });
 
-            pac.stdout.on('data', (data) => stdout.push(...data.toString().split(os.EOL)));
-            pac.stderr.on('data', (data) => stderr.push(...data.toString().split(os.EOL)));
+            process.stdout.on('data', (data) => stdout.push(...data.toString().split(os.EOL)));
+            process.stderr.on('data', (data) => stderr.push(...data.toString().split(os.EOL)));
 
-            pac.on('close', (code) => {
+            process.on('exit', (code) => {
                 if (code === 0) {
                     this.logger.info(`success: ${stdout.join(os.EOL)}`);
                     resolve(stdout);
+
+                    // Close out handles to the output streams so that we don't wait on grandchild processes like pacTelemetryUpload
+                    process.stdout.destroy();
```

Comment:
move the common code to after the if/else block, since the handle closures are now the same

Reply:
moved